packages/core/test/temporal/NativeJsTemporalTest.js

```diff
@@ -19,102 +19,182 @@ import { ZonedDateTime } from '../../src/ZonedDateTime';
+            ChronoField.MICRO_OF_DAY,
+            ChronoField.MILLI_OF_SECOND,
+            ChronoField.MILLI_OF_DAY,
+            ChronoField.SECOND_OF_MINUTE,
+            ChronoField.SECOND_OF_DAY,
+            ChronoField.MINUTE_OF_HOUR,
+            ChronoField.MINUTE_OF_DAY,
+            ChronoField.HOUR_OF_AMPM,
+            ChronoField.CLOCK_HOUR_OF_AMPM,
+            ChronoField.HOUR_OF_DAY,
+            ChronoField.CLOCK_HOUR_OF_DAY,
+            ChronoField.AMPM_OF_DAY,
+            ChronoField.DAY_OF_WEEK,
+            ChronoField.ALIGNED_DAY_OF_WEEK_IN_MONTH,
+            ChronoField.ALIGNED_DAY_OF_WEEK_IN_YEAR,
+            ChronoField.DAY_OF_MONTH,
+            ChronoField.DAY_OF_YEAR,
+            ChronoField.EPOCH_DAY,
+            ChronoField.ALIGNED_WEEK_OF_MONTH,
+            ChronoField.ALIGNED_WEEK_OF_YEAR,
+            ChronoField.MONTH_OF_YEAR,
+            ChronoField.PROLEPTIC_MONTH,
+            ChronoField.YEAR_OF_ERA,
+            ChronoField.YEAR,
+            ChronoField.ERA,
+            ChronoField.INSTANT_SECONDS,
+            ChronoField.OFFSET_SECONDS,
+        ];
+        for (const temporalField of testData) {
+            assertTrue(temporalAccessor.isSupported(temporalField));
+        }
+    });
+
+    it('should support all iso fields', () => {
+        const temporalAccessor = nativeJs(new Date(0));
+        const testData = [
+            IsoFields.DAY_OF_QUARTER,
+            IsoFields.QUARTER_OF_YEAR,
+            IsoFields.WEEK_OF_WEEK_BASED_YEAR,
+            IsoFields.WEEK_BASED_YEAR,
+        ];
+        for (const temporalField of testData) {
+            assertTrue(temporalAccessor.isSupported(temporalField));
+        }
     });
 
-});
\ No newline at end of file
+    it('should support no chrono units', () => {
+        const temporalAccessor = nativeJs(new Date(0));
+        const testData = [
+            ChronoUnit.NANOS,
+            ChronoUnit.MICROS,
+            ChronoUnit.MILLIS,
+            ChronoUnit.SECONDS,
+            ChronoUnit.MINUTES,
+            ChronoUnit.HOURS,
+            ChronoUnit.HALF_DAYS,
+            ChronoUnit.DAYS,
+            ChronoUnit.WEEKS,
+            ChronoUnit.MONTHS,
+            ChronoUnit.YEARS,
+            ChronoUnit.DECADES,
+            ChronoUnit.CENTURIES,
+            ChronoUnit.MILLENNIA,
+            ChronoUnit.ERAS,
+            ChronoUnit.FOREVER,
+        ];
+        for (const temporalField of testData) {
+            assertThrows(IllegalArgumentException, () => temporalAccessor.isSupported(temporalField));
+        }
+    });
+
+    it('should support no iso units', () => {
+        const temporalAccessor = nativeJs(new Date(0));
+        const testData = [
+            IsoFields.WEEK_BASED_YEARS,
+            IsoFields.QUARTER_YEARS,
+        ];
+        for (const temporalField of testData) {
+            assertThrows(IllegalArgumentException, () => temporalAccessor.isSupported(temporalField));
+        }
+    });
+});
```

Comment:
What about adding a test that shows / explains the benefit of your PR, why not te add a test for  

```
DayOfWeek.from(nativeJs(date));
Year.from(nativeJs(date));
Month.from(nativeJs(date));
YearMonth.from(nativeJs(date));
// and maybe more 
```

Reply:
There are 2 such tests ("should support all chrono fields", "should support all iso fields"). It's impossible to list all factory methods taking `TemporalAccessor` that are fixed with this PR, that why I based my tests on `TemporalField`s.